### PR TITLE
Feature: embed form within a page

### PR DESCRIPTION
**Pull request by shaheertarar**
"embedded" viewMode and a selector can be passed as options to create form within given element

**Changes to formhero.js**
- changed formData: formData to formData: prepopulatedData
- Error handling if user passes "embedded" viewMode without passing a selector as an option
- new loadFormEmbedded function

**Changes to formhero.scss** 
- new embedded class

### CODE REVIEW

File: src/formhero.js [hunk 1]
```diff
@@ -391,27 +393,69 @@ var formhero = (function (api) {
...
-    {
+    function loadFormInPage(formUrl, options){
         document.location = formUrl;
     }
+    
+    function loadFormEmbedded(formUrl, formFrameIdentifier, options){
+        console.log('FormHero Embedded');
+        formUrl += '&iframeId=' + formFrameIdentifier;
+        
+        if(true) { //else {
```
Comment (jerrieyang) [inline src/formhero.js]: if(true) ?

**Comment by shaheertarar** [comment]
removed if(true)

File: src/formhero.js [hunk 2]
```diff
@@ -391,27 +392,67 @@ var formhero = (function (api) {
...
+
+        var iframeContainer = document.createElement('div');
+        iframeContainer.id = 'formhero-iframe-wrapper-' + formCount;
+        iframeContainer.className = 'formhero-iframe-wrapper visible embedded';
+
+        var iframe = document.createElement('iframe');
+        iframe.className = 'formhero-iframe embedded';
+        iframe.id = 'form-frame-' + formCount;
+        iframe.name = 'form-frame-' + formCount;
+        iframe.src = formUrl + '&viewMode=embedded';
```
Comment (jerrieyang) [inline src/formhero.js]: please remove `'&viewMode=embedded';`, the filler will default it to 'window'

Comment (shaheertarar) [reply src/formhero.js]: removed

**Pull request merged**